deps/spidershim/test/v8engine.h

```diff
@@ -95,6 +95,14 @@ class V8Engine {
     return Local<Value>();
   }
 
+  Local<Value> CompileRun(Local<Context> context, Local<String>* script) {
```

Comment:
> Also it would be nice if you refactored the other CompileRun overload to be implemented in terms of this using v8_str or something like that...

Yup, makes sense, done in eaa88ed.